seqr/views/utils/orm_to_json_utils.py

```diff
@@ -204,7 +204,7 @@ def _get_pedigree_image_url(pedigree_image):
                 pedigree_image = pedigree_image.url
             except Exception:
                 pedigree_image = None
-        return os.path.join("/media/", pedigree_image) if pedigree_image else None
+        return pedigree_image
```

Comment:
🤦

Reply:
are we sure this works if you have local image files?

Reply:
I'll double check this, but I tested the code on dev with the GCS stuff turned on and off, and we were getting correct URL's in both cases. My understanding is that the `.url` attribute will prepend the media path onto the image name when returning the value. My belief in this case was that `pedigree_image.url` was returning something like `/media/foo.png`, and then we were doing a no-op operation with `os.path.join`, like this:

```
>>> test_path = "/media/foo.png"
>>> import os
>>> os.path.join("/media/", test_path)
'/media/foo.png'
```

Reply:
resolving this, as it was handled in the GCR media repo PR